cfgov/unprocessed/css/forms.less

```diff
@@ -607,44 +398,346 @@
+      - "You MUST wrap your input.cf-input in a label."
+      - "Remember to use 'for' and 'id' attributes so that selected/checked styles function properly."
+      - "Remember to use the same 'name' attribute for each set of radio buttons."
+    patterns:
+      - name: Checkbox
+        markup: |
+          <label class="cf-input_label">
+              <input class="cf-input cf-input_checkbox" type="checkbox" name="option1" value="Option 1">
+              Option 1
+          </label>
+          <br>
+          <label class="cf-input_label">
+              <input class="cf-input cf-input_checkbox" type="checkbox" name="option2" value="Option 2">
+              Option 2
+          </label>
+        codenotes:
+          - |
+            Structural cheat sheet:
+            -----------------------
+            label.cf-input_label
+              input[type=checkbox].cf-input
+        notes:
+          - "Use the markup structure above to add checkbox .cf-input elements."
+      - name: Radio buttons
+        markup: |
+          <label class="cf-input_label" for"yes">
+              <span class="cf-input_text">Yes</span>
+              <input class="cf-input cf-input_hidden" type="radio" name="cf-input-example_default" id="yes">
+              <span class="cf-input_radio"></span>
+          </label>
+          <br>
+          <label class="cf-input_label" for="no">
+              <span class="cf-input_text">No</span>
+              <input class="cf-input cf-input_hidden" type="radio" name="cf-input-example_default" id="no">
+              <span class="cf-input_radio"></span>
+          </label>
+        codenotes:
+          - |
+            Structural cheat sheet:
+            -----------------------
+            label.cf-input_label
+              span.cf-input_text
+              input[type=radio].cf-input.cf-input_hidden
+              span.cf-input_radio
+        notes:
+          - "Use the markup structure above to add radio button .cf-input elements."
+          - "Radio buttons require extra markup to apply custom styles."
+    tags:
+      - cf-forms
+*/
+
+@input-active:                  @pacific;
+
+@input-bg-disabled:             @gray;
+@input-border-disabled:         @gray-50;
+
+// Insets
+@input-inset-bg:                @gray-20;
+@input-inset-selected:          @pacific-20;
+
+
+input.cf-input {
+
+    @checkbox-width: unit( 18px / @base-font-size-px, em);
+    @radio-width: unit( 20px / @base-font-size-px, em);
+
+    &[type="checkbox"],
+    &[type="radio"]{
+      .u-visually-hidden();
+
+      .lt-ie9 & {
+        border: 0;
+        float: left;
+        margin: .4em .4em 0 0;
+        position: static;
+        width: auto;
+      }
+    }
+
+    &[type="checkbox"] + label,
+    &[type="radio"] + label {
+      cursor: pointer;
+      font-weight: 400;
+      margin-bottom: 0.5em;
+    }
+
+    &[type="checkbox"] + label::before,
+    &[type="radio"] + label::before {
+      background: white;
+      border-radius: 4px;
```

Comment:
border-radius and box-shadow should be moved to L:551 to avoid overwriting unnecessarily